ldms/src/ldmsd/ldmsd_request.c

```diff
@@ -1729,11 +1729,17 @@ static int prdcr_start_handler(ldmsd_req_ctxt_t reqc)
 
 	ldmsd_req_ctxt_sec_get(reqc, &sctxt);
 	interval_str = ldmsd_req_attr_str_value_get_by_id(reqc,
-							LDMSD_ATTR_INTERVAL);
+							LDMSD_ATTR_RECONNECT);
```

Comment:
Why do we need a new attribute?

Reply:
The only reason I added `LDMSD_ATTR_RECONNECT` is to differentiate between the uses of 'interval' and 'reconnect' in configuration files, so LDMSD could give a warning message that the 'interval' attribute is being deprecated.

Reply:
Hi @nichamon, I understand, but my suggestion is that the server shouldn't do this, ldmsd_controller, ldmsdctl and the config parsing should.

Reply:
I pushed a patch that gets rid of the new attribute `LDMSD_ATTR_RECONNECT` and let the parsers for ldmsd_controller, ldmsctl, and config files give the warning about 'interval' being deprecated.